CMake/SlicerBlockFindQtAndCheckVersion.cmake

```diff
@@ -72,16 +85,23 @@ if(NOT Slicer_SOURCE_DIR)
 endif()
 message(STATUS "Configuring ${_project_name} with Qt ${_major}.${_minor}.${_patch} (using modules: ${command_separated_module_list})")
 
-# Since Qt5 does not set CMake variables for plugins and binary directories,
+# Since neither Qt5 or Qt6 set CMake variables for plugins and binary directories,
 # we explicitly set them here.
-  set(QT_PLUGINS_DIR "${Qt5_DIR}/../../../plugins")
-  get_filename_component(QT_PLUGINS_DIR ${QT_PLUGINS_DIR} ABSOLUTE)
-  message(STATUS "Setting QT_PLUGINS_DIR: ${QT_PLUGINS_DIR}")
+set(QT_PLUGINS_DIR "${Qt${_major}_DIR}/../../../plugins")
+get_filename_component(QT_PLUGINS_DIR ${QT_PLUGINS_DIR} ABSOLUTE)
+message(STATUS "Setting QT_PLUGINS_DIR: ${QT_PLUGINS_DIR}")
 
-  set(QT_BINARY_DIR "${Qt5_DIR}/../../../bin")
-  get_filename_component(QT_BINARY_DIR ${QT_BINARY_DIR} ABSOLUTE)
-  message(STATUS "Setting QT_BINARY_DIR: ${QT_BINARY_DIR}")
+set(QT_BINARY_DIR "${Qt${_major}_DIR}/../../../bin")
+get_filename_component(QT_BINARY_DIR ${QT_BINARY_DIR} ABSOLUTE)
+message(STATUS "Setting QT_BINARY_DIR: ${QT_BINARY_DIR}")
 
-set(QT_LIBRARY_DIR "${Qt5_DIR}/../../../lib")
+set(QT_LIBRARY_DIR "${Qt${_major}_DIR}/../../../lib")
```

Comment:
https://github.com/Slicer/Slicer/pull/8825/files#r2516716666

Reply:
Hope I have some time later today or this week to look into this again on Fedora - there should be a more generic way to retrieve the plugins folder, probably [through target properties of imported plugins target](https://stackoverflow.com/questions/14183441/get-path-of-qt-5-additional-plugins).

Edit: Wait a minute - previous discussion was about `QT_PLUGINS_DIR` (line 90, above), not `QT_LIBRARY_DIR`...?

Reply:
@lassoan @codeling anyway, currently setting in ArchLinux:
```
-- Setting QT_PLUGINS_DIR: /usr/plugins
-- Setting QT_BINARY_DIR: /usr/bin
-- Setting QT_LIBRARY_DIR: /usr/lib
```
but according to [extra/x86_64/qt6-base/](https://archlinux.org/packages/extra/x86_64/qt6-base/), `QT_PLUGINS_DIR` should be `/usr/lib/qt6/plugins/`.